base/exports.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 export
-# Modules
+    # Modules
```

Comment:
Please do not change formatting unecessarily

Reply:
that was unintentional -- thank you for catching the mistype